domain/src/main/java/com/harman/domain/model/RandomNumber.kt

```diff
@@ -0,0 +1,3 @@
+package com.harman.domain.model
```

Comment:
now it seems a little bit strange, but lets keep domain, data and app modules in your `package="com.harman.roomdbapp"`, so this will be like: `com.harman.roomdbapp.domain.model` and please change another modules too.

Reply:
@AlexeyKolosov, you mean to just change package name in modules?

Reply:
Yes, seemed that directory was wrong, for now it is good for domain module, please check other modules also